sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -188,11 +200,60 @@ export class IdentityClient extends ServiceClient implements INetworkModule {
     }
   }
 
+  // Here is a custom layer that allows us to abort requests that go through MSAL,
+  // since MSAL doesn't allow us to pass options all the way through.
+
+  private abortControllers?: Record<string, AbortController[]>;
+
+  generateAbortSignal(correlationId?: string): AbortSignalLike {
+    if (!this.abortControllers) {
```

Comment:
why lazily initialize this?

Reply:
I'll push that in my next commit, so I'll resolve this to help me focus on what's left.